Dockerfile

```diff
@@ -16,13 +15,9 @@ WORKDIR $HOME/app
 
 COPY --chown=user . $HOME/app
 
-RUN ollama serve & \
-    sleep 5 && \
-    ollama create volo -f ./Modelfile
-
 RUN python3 -m pip install --no-cache-dir -r requirements.txt
 
 EXPOSE 7860
 
 ENTRYPOINT []
-CMD ["bash", "-c", "/bin/ollama serve & chainlit run app.py -h -d --port 7860"]
+CMD ["/bin/bash", "-c", "/bin/ollama create volo -f Modelfile && chainlit run app.py -h -d --port 7860"]

```

Comment:
Instead of downloading the model into the build layers of the container, I instead download it at runtime, ensuring the latest model each time the server starts up and lowering the size of the container, which is a factor with llama3.1